.vscode/tasks.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I'm ok with devcontainer. For .vscode my suggestion is against this. .vscode is a very opinionated per devs thing and for a open source project it should definitely be agnostic about all of this.

Though it is nice, often people would have more stuff in here, or custom tasks, and might accidentally check this in. 

my philosophy with these config is if you need to use it you will probably be able to google how to.

Reply:
I'm ok if we decided to split out/remove the VSCode bits. My goal is to make the new user dev experience easier for less technical users.

> vscode is a very opinionated per devs thing

Because the existence of the file doesn't affect any other editor it don't believe it negates the potential value. It does create an easier developer experience for users who are new. Again, not a hill I'll die on.